tests/contrib/operators/test_gcs_to_gcs_operator.py

```diff
@@ -41,20 +41,24 @@
 DESTINATION_OBJECT_PREFIX = 'foo/bar'
 SOURCE_FILES_LIST = [
     'test_object/file1.txt',
-    'test_object/file2.txt',
-    'some_other/file.txt'
```

Comment:
This and the next two lines are fixes for unrelated errors in the previous test (also added by me).